docs/_data/components/schemas/headers.yaml

```diff
@@ -57,9 +57,28 @@ forecast-proposal-header:
     - default-emergency-durations
     - power-system-resources
 
-forecast-snapshot-header:
-  description: Details about the snapshot provided by the Clearinghouse provider.
+forecast-proposal-slim-header:
+  <<: *forecast-proposal-header
   properties:
+    <<: *forecast-proposal-header-props
+    ends:
```

Comment:
Didn't we also need to have something in here specifying rating units?

Reply:
That is where the "esoteric" use of media type parameters came in. :)
https://trolie.energy/articles/media-types.html#curl-example

Reply:
Oh yeah I forgot.